src/main/java/frc/robot/subsystems/drivetrain/DrivetrainImpl.java

```diff
@@ -44,9 +48,28 @@ public DrivetrainImpl(){
         this.rightMotorControllers = new SpeedControllerGroup(this.rightMotorController1, this.rightMotorController2, this.rightMotorController3);
         this.rightEncoder = new Encoder(RobotMap.RIGHT_ENCODER_PORTS[0], RobotMap.RIGHT_ENCODER_PORTS[1]);
 
+        this.leftPIDController = new PIDController(RobotMap.Kp, RobotMap.Ki, RobotMap.Kd); // TODO setup SmartdashboardPIDTuner
+        this.rightPIDController = new PIDController(RobotMap.Kp, RobotMap.Ki, RobotMap.Kd);
```

Comment:
These constants should at least be named `DRIVETRAIN_KP` rather than just `Kp` if they're going to live in the global RobotMap, but even putting them into the RobotMap seems weird to me. These are very clearly specific to the DrivetrainImpl and it would be super weird if anything else needed access to them.

Reply:
I just put them in RobotMap so when adjusting them they are in a more accessible location. Renaming them definitely is a good idea though.

Reply:
I don't really see how RobotMap is "more accessible". It's just a file to open one way or the other. At least if they're in DrivetrainImpl then they're in the same place they're used, so you only have one file open.

Reply:
Yeah that's true. I was expecting us to have multiple PID loops this year when I initially put that there and thought it would be handy to be able to tune them all from one place.